### ISSUE TITLE
Reset cache lifetime and driver after query execution (?)

### ISSUE THREAD

**Issue by rsdev000** [rinvex/repository#31]
Steps to reproduce

Execute two or more queries in the same request.

```php
$x = $this->repository->setCacheDriver('redis')->findWhere(['post_id' => 10]);
$y = $this->repository->findWhere(['post_id' => 20]);
```

Both queries are cached in redis but the second query doesn't say anything about the cache driver so it's expected to be cached in the default driver and with the default lifetime (?)

Cheers

**Comment by Omranic** [comment]
Yeah, seems the same cause of the previous issue #30 
I guess the natural & logical way of thinking is to reset everything on the repository after a query retrieval. Working on it.. Let me know if you've any other thoughts :)

**Comment by Omranic** [comment]
Fixed, check latest release **v2.0.0** 👍
